integration-examples/splunk-otel-databricks/Splunk OpenTelemetry Collector Install.md

```diff
@@ -0,0 +1,120 @@
+# Splunk OpenTelemetry Collector Installation on Databricks
+
+## OpenTelemetry Collector Installation
+
+There are multiple ways to install the splunk opentelemetry collector on databricks and the host infrastructure. An example start-up script has been provided where Splunk Observability Cloud credentials are saved in the Databricks secret store.
+
+The `splunk-start-otel.sh` script will execute successfully if using paid editions but not on the community edition of Databricks. This script only installs the default collector without Apache Spark metrics collection enabled. To enable this metric collection the agent yaml file will need to modified for the controller or workers.
```

Comment:
is there any link to the yaml that would need to be modified or docs on what to add or anything from our docs?

Reply:
Or maybe a link to the heading below in the docs where you cover this (that I hadn't gotten to yet)

Reply:
I can add a link to the documentation for the start-up script and it's options. That will be a great way to clarify the additional options.

Reply:
https://github.com/splunk/observability-content-contrib/pull/38/commits/a10b300e33dd32aff1db6df1dd28a8e94461fcb3 this commit is adding additional references.